### PR TITLE
Feat upgrade search

### PR DESCRIPTION
**Pull request by amanda-yosh**
#### Solution
Upgrade the search bar component so that the user can look for the city and business type of interest.

#### What issue action is associated with this pull request?
None.
#### What changes were proposed in this pull request?
The user will have a more effective search.

### CODE REVIEW

File: src/api/glue-api.js [hunk 1]
```diff
@@ -3,13 +3,12 @@ import { BASE_URL } from '../settings/constants'
 
 const resultPage = '/listings?categoryPage=RESULT'
 const listingPage = '/page'
-const villageHouse = '&unitSubTypes=VILLAGE_HOUSE&unitTypes=HOME&usageTypes=RESIDENTIAL&unitTypesV3=VILLAGE_HOUSE'
+const villageHouse = '&unitSubTypes=VILLAGE_HOUSE&unitTypes=HOME&usageTypes=RESIDENTIAL&unitTypesV3=VILLAGE_HOUSE&text=Casa+de+Vila'
```
Comment (cmarcal) [inline src/api/glue-api.js]: You could create a function to mount theses query params, so you not necessary crate one by one. What do you think?

File: src/helpers/utils.js [hunk 2]
```diff
@@ -1,14 +1,9 @@
 import { router } from '../router'
```
Comment (cmarcal) [inline src/helpers/utils.js:1]: Why you not create test for theses cases ?

File: src/store/listing/getters.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
-const getHrefFromId = (state) => (id) => {
-  return state.href.get(id)
-}
+const getHrefFromId = (state) => (id) => { return state.href.get(id) }
```
Comment (cmarcal) [inline src/store/listing/getters.js]: You don't need this sintax if you return in same line. 
You can as well as this exemple: 

`const getHrefFromId = (state) => (id) => state.href.get(id)`

What do you think ?

File: src/store/listing/getters.js [hunk 4]
```diff
@@ -1,7 +1,8 @@
...
+
+const getListings = (state) => { return state.listings ?? [] }
```
Comment (cmarcal) [inline src/store/listing/getters.js]: same case here

File: src/helpers/utils.js (hunk 2 above)
Comment (amanda-yosh) [reply src/helpers/utils.js:1]: Im waiting this branch to be merged for create the tests on this other branch [feat-create-tests](https://github.com/amanda-yosh/the-shire/pull/9)

File: src/store/listing/getters.js (hunk 3 above)
Comment (amanda-yosh) [reply src/store/listing/getters.js]: I agree, done!

File: src/store/listing/getters.js (hunk 4 above)
Comment (amanda-yosh) [reply src/store/listing/getters.js]: done!

File: src/api/glue-api.js (hunk 1 above)
Comment (amanda-yosh) [reply src/api/glue-api.js]: Ive created a function, but I still have to pass the params for now.

**Pull request merged**